lib/models/living.py

```diff
@@ -0,0 +1,48 @@
+from dataclasses import dataclass
+from typing import List, Dict
+
+from lib.models.character_class import CharacterClass
+from lib.models.entity import *
+from lib.models.enums import Ability, Skill, Alignment
+
+
+@dataclass
+class Living(Entity):
+    name: str
+    description: str
+    character_class: CharacterClass 
+    level: int
+    background: str
+    race: str
+    alignment: Alignment
+    xp: int
+    abilities: Dict[Ability, int]
+    skills: List[Skill]
+    max_hp: int
+    current_hp: int
+    temporary_hp: int
+    armor_class: int
+    hd_value: int
+    hd_total: int
+    death_save_success: int = 0
+    death_save_failure: int = 0
+    inventory: Inventory = Inventory()
+
+    def get_modifier(self, ability: Ability) -> int:
```

Comment:
Could be a bit of metaprogramming to have a method for each ability here, so you have the nicer interface of `get_strength_modifier` instead of `get_modifier(Ability.STRENGTH)`

Reply:
My thinking behind this is if you take a look at the Skill enum, you can just use the Ability value from the tuple to get the modifier, rather than having to figure out what the Ability is first, then call the appropriate method.

Reply:
Maybe both interfaces are useful. One for input, one for human written methods

Reply:
Klingt gut.

Reply:
Something for a future PR though